editor/editor_node.cpp

```diff
@@ -1735,6 +1735,11 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			file->set_title(p_option == FILE_OPEN_SCENE ? TTR("Open Scene") : TTR("Open Base Scene"));
 			file->popup_centered_ratio();
 
+		} break;
+		case FILE_OPEN_MAIN_SCENE: {
+
+			load_scene(ProjectSettings::get_singleton()->get("application/run/main_scene"));
```

Comment:
And what if no main scene is defined? :)

Reply:
Oops..
How it would look better? Just show an error message or ask if you want to set a main scene (same as "Play the project")?

Reply:
Maybe don't add the option to the menu if no main scene is defined?